### PR TITLE
Add privacy policy

### LINKED ISSUE #158: Adding a privat policy information
I read from time to time about people complaining about Solus not having a Private Policy, I looked around and found a comment by Ikey on reddit [Link](https://www.reddit.com/r/SolusProject/comments/6c50vd/where_can_i_find_the_privacy_policy_for_the_solus/?st=j3gmgab5&sh=5dcf0b87)

- The server logs are automatically rotated, and are only HTTP access logs like any web server. They're not used for analytics and nuke themselves over time.
- We don't collect any data.
- We use cookies on some of the websites to make login work. Some third party mechanisms may store their own cookies (such as the antispam measures provided by Google)
- No information is collected from your OS
- Some applications may have their own in-built telemetry, such as Firefox. This goes to Mozilla and Solus has no involvement in it. You may also turn this off.

This can certainly expand and explained in detail

Dunno where something like this could be added inside the help-center-docs, so I didn't add any PR :)

### PR DESCRIPTION
**Pull request by davidjharder**
## Description

Add a privacy policy. *This is a draft*

Resolves #158

### Submitter Checklist

- [x] Squashed commits with `git rebase -i` (if needed)

### CODE REVIEW

File: docs/user/privacy.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+#### Matrix
+
+- The Solus matrix rooms are hosted by the Matrix.org homeserver
+- [Matrix.org Homeserver Privacy Notice](https://matrix.org/legal/privacy-notice) 
+
+
+## Your rights under the GDPR
+
+If you are covered by the [GDPR](https://gdpr.eu/) you have rights as a data subject. Your rights are detailed under [Chapter 3 of the GDPR](https://gdpr.eu/article-12-how-controllers-should-provide-personal-data-to-the-subject/)
```
Comment (silkeh) [inline docs/user/privacy.md:56]: I feel like there should be a more user-friendly link for this, but I wasn't able to quickly find one from an EU website :disappointed:

**Review by silkeh — CHANGES_REQUESTED** [review]
Generally looks good, clear and to the point!

File: docs/user/privacy.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Privacy Policy  (Draft)
+
+## Solus Installations
+
+- Solus does not collect or store data from user's machines.
+- Solus does not use telemetry or automatic error reporting.
+- Some packages may include their own built-in telemetry (examples: Firefox, Steam). Solus cannot control data collected in this way. These packages may allow you to disable this telemetry.
```
[the remark this answers is not in the document]
Comment (davidjharder) [reply docs/user/privacy.md:7]: What we do to individual packages does not change this wording unless we are certain we have disabled all telemetry everywhere. Maybe this is a good goal, but for now this wording is accurate

File: docs/user/privacy.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  - Name or Username
+  - Email address
+  - Timezone (optional)
+  - SSH key (optional)
+- If you decide to contribute to packaging, your **name** and **email address** will be publicly viewable. This information will be attached to your contribution, even if you delete your account, based on our interest in maintaining documentation on submitted packages.
+- If you decide to submit bug reports or other issues, then delete your account, we will keep your submitted issues based on our interest in maintaining consistent bug reporting and issue tracking, but your name will be removed.
+
+### Forum
+
+- `discuss.getsol.us`
```
[the remark this answers is not in the document]
Comment (davidjharder) [reply docs/user/privacy.md:11]: done

File: docs/user/privacy.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Privacy Policy  (Draft)
```
Comment (EbonJaeger) [inline docs/user/privacy.md:1]: Might want to remove that before merging :P

```suggestion
# Privacy Policy
```

**Review by EbonJaeger — CHANGES_REQUESTED** [review]
Just one minor thing.

File: docs/user/privacy.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+- [Matrix.org Homeserver Privacy Notice](https://matrix.org/legal/privacy-notice) 
+
+
+## Your rights under the GDPR
+
+If you are covered by the [GDPR](https://gdpr.eu/) you have rights as a data subject. Your rights are detailed under [Chapter 3 of the GDPR](https://gdpr.eu/article-12-how-controllers-should-provide-personal-data-to-the-subject/) Additionally, you have the right to lodge a complaint with your national [Data Protection Authority](https://edpb.europa.eu/about-edpb/board/members_en).
+
+## Privacy Contact
+
+Any concerns or requests about personal data should be sent to [privacy@getsol.us](mailto:privacy@getsol.us) TODO: what is the correct email?
```
Comment (davidjharder) [inline docs/user/privacy.md:46]: Also, this is draft language still

File: docs/user/privacy.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+#### Matrix
+
+- The Solus matrix rooms are hosted by the Matrix.org homeserver
+- [Matrix.org Homeserver Privacy Notice](https://matrix.org/legal/privacy-notice) 
+
+
+## Your rights under the GDPR
+
+If you are covered by the [GDPR](https://gdpr.eu/) you have rights as a data subject. Your rights are detailed under [Chapter 3 of the GDPR](https://gdpr.eu/article-12-how-controllers-should-provide-personal-data-to-the-subject/) Additionally, you have the right to lodge a complaint with your national [Data Protection Authority](https://edpb.europa.eu/about-edpb/board/members_en).
```
Comment (silkeh) [inline docs/user/privacy.md:42]:
```suggestion
If you are covered by the [GDPR](https://gdpr.eu/) you have rights as a data subject. Your rights are detailed under [Chapter 3 of the GDPR](https://gdpr.eu/article-12-how-controllers-should-provide-personal-data-to-the-subject/). Additionally, you have the right to lodge a complaint with your national [Data Protection Authority](https://edpb.europa.eu/about-edpb/board/members_en).
```

**Review by silkeh — APPROVED** [review]

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks! :)

**Pull request merged**